src/App.js

```diff
@@ -70,4 +69,25 @@ class App extends React.Component {
         );
     }
 }
-export default App;
+
+App.propTypes = {
+    state: PropTypes.object.isRequired,
+    actions: PropTypes.object.isRequired,
+};
+
+function mapStateToProps(state) {
+    return {
+        state,
```

Comment:
This is not very useful. The idea of mapStateToProps is to filter the state and pass on to the component only what you need, not the whole state object.